pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/admin/UpdateUserRoleRequest.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.dsm.service.admin;
+
+import java.util.List;
+
+import lombok.Data;
+
+@Data
+public class UpdateUserRoleRequest {
+
+    private final List<String> users;
+    private final List<String> addRoles;
+    private final List<String> removeRoles;
+
+    public UpdateUserRoleRequest(List<String> users, List<String> addRoles, List<String> removeRoles) {
```

Comment:
Is this class to add and remove all the roles to ALL the users ?
I mean ALL users have same exact changes (add and remove passed roles), right ?

Reply:
That is correct. It is unlikely the UI will use this, at least in the first iteration, but this is useful for bulk operations so I kept it around. (I actually used the former version of the associated endpoint for many of the permission updates over the past couple sprints.)